timetracker/vms/test/models/test_time_record_model.py

```diff
@@ -17,3 +20,40 @@ def test_is_approved_with_approval(time_record_approval_factory):
     record = approval.time_record
 
     assert record.is_approved
+
+
+def test_1_time_string_conversion(time_record_factory):
+    """
+    Case with no end time
+    """
+    t = time_record_factory()
+    expected = (
+        f'Time Record starting at {t.start_time:%I:%M %p}'
+        f' on {t.start_time:%m/%d/%Y}.'
+    )
+    assert str(t) == expected
+
+
+def test_2_time_string_conversion(time_record_factory):
+    """
+    Case with different end times
+    """
+    t = time_record_factory(end_time=datetime.datetime())
+    expected = (
+        f'Time Record from {t.start_time:%I:%M %p on %m/%d/%Y}, '
+        f'{t.end_time:%I:%M %p on %m/%d/%Y}.'
+    )
+    assert str(t) == expected
+
+
+def test_3_time_string_conversion(time_record_factory):
+    """
+    Case with same end times
+    """
+    d = datetime.datetime()
```

Comment:
You have to provide some sort of date information when constructing a `datetime` instance. If you want the current time, you can do `datetime.datetime.now()`, or you can look at [the `datetime` documentation](https://docs.python.org/3/library/datetime.html#datetime-objects) to see the information required to instantiate a `datetime` instance.